help/sources/tutorials/ui/create/advertising/pinterest-ads.md

```diff
@@ -0,0 +1,264 @@
+
+Finally, you must provide as a mandatory input the field values below:
+| Field | Description |
+| --- | --- |
+| **[!UICONTROL ad_account_id]** | Your [!DNL Pinterest Ads] ad_account_id. You can refer to the [Find IDs in Ads Manager](https://help.pinterest.com/en/business/article/find-ids-in-ads-manager) page for guidance on how to get the value. |
+| **[!UICONTROL object_type]** | Select one of **Campaigns**, **Ad Groups** or **Ads** depending on which of the [!DNL Pinterest] Analytics APIs you want to obtain information from. |
+| **[!UICONTROL object_ids]** | The selected object's IDs. Navigate to the [!DNL Pinterest] page for **Pinterest Business Hub** > **Ad Account Summary** > **Campaigns** / **Ad Groups** / **Ads** and copy the required ID's mentioned just below each of their names. |
+
+>[!NOTE]
+>
+> Multiple **[!UICONTROL object_ids]** can be provided by passing comma separated values.
+> A maximum of 100 IDs can be passed at a time.
+> If incorrect values are passed platform will show an error prompt with the message below :
+> `The request could not be processed. Error from flow provider: Unknown error while processing request.` 
+
+After providing the values, select the **[!UICONTROL Select]**. If the values provided are valid, the right part of the interface, preview data, will get populated.
+
+![The select data step of the sources workflow.](../../../../images/tutorials/create/advertising/pinterest-ads/select-data.png?lang=en)
+
+## Dataflow detail {#dataflow-detail}
+
+The **Dataflow detail** step appears, providing you with options to use an existing dataset or establish a new dataset for your dataflow, as well as an opportunity to provide a name and description for your dataflow. During this step, you can also configure settings for Profile ingestion, error diagnostics, partial ingestion, and alerts.
+
+When finished, select **[!UICONTROL Next]**.
+
+![The dataflow-detail step of the sources workflow.](../../../../images/tutorials/create/advertising/pinterest-ads/dataflow-detail.png?lang=en)
+
+## Mapping {#mapping}
+
+The [!UICONTROL Mapping] step appears, providing you with an interface to map the source fields from your source schema to their appropriate target XDM fields in the target schema.
+
+Platform provides intelligent recommendations for auto-mapped fields based on the target schema or dataset that you selected. You can manually adjust mapping rules to suit your use cases. Based on your needs, you can choose to map fields directly, or use data prep functions to transform source data to derive computed or calculated values. For comprehensive steps on using the mapper interface and calculated fields, see the [Data Prep UI guide](../../../../../data-prep/ui/mapping.md).
+
+Once your source data is successfully mapped, select **[!UICONTROL Next]**.
+
+![The mapping step of the sources workflow.](../../../../images/tutorials/create/advertising/pinterest-ads/mapping.png)
+
+## Scheduling {#scheduling}
```

Comment:
lets move this to additional resources as it is specific to pinterest and has some additional info compared to the modularized scheduling step in the main advertising dataflow tutorial